### PR TITLE
Add confirm box in infraenv delete modal

### PR DESCRIPTION
**Pull request by ammont82**
Issue related: https://bugzilla.redhat.com/show_bug.cgi?id=2056472

Signed-off-by: Montse Ortega <mortegag@redhat.com>

### CODE REVIEW

File: frontend/src/routes/Infrastructure/InfraEnvironments/InfraEnvironmentsPage.tsx [hunk 1]
```diff
@@ -210,7 +210,7 @@ const InfraEnvsTable: React.FC<InfraEnvsTableProps> = ({ infraEnvs, agents }) =>
                                                 setModalProps({ open: false })
                                             },
                                             isDanger: true,
-                                            icon: 'warning',
+                                            icon: 'warning',
```
Comment (rawagner) [inline frontend/src/routes/Infrastructure/InfraEnvironments/InfraEnvironmentsPage.tsx:213]: can you remove the extra characters ?

Comment (ammont82) [reply frontend/src/routes/Infrastructure/InfraEnvironments/InfraEnvironmentsPage.tsx]: done!

**Pull request merged**